### PR TITLE
cn: added node/cn/peer_test.go

### PR DESCRIPTION
**Pull request by ehnuje**
## Proposed changes

- added node/cn/peer_test.go to test cn.Peer interface

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [ ] New feature or enhancement
- [x] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [x] Lint and unit tests pass locally with my changes (`$ make test`)
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

- Please leave the issue numbers or links related to this PR here.

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: node/cn/peer_test.go [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+		if err := basePeer.Send(NewBlockHashesMsg, data); err != nil {
+			t.Fatal(err)
+		}
+	}(t)
+	receivedMsg, err := oppositePipe.ReadMsg()
+	if err != nil {
+		t.Fatal(err)
+	}
+	assert.Equal(t, expectedMsg.Code, receivedMsg.Code)
+	assert.Equal(t, expectedMsg.Size, receivedMsg.Size)
```
Comment (jeonghonah) [inline node/cn/peer_test.go]: How about compare payload also?

https://github.com/klaytn/klaytn/blob/fc83163bf5f18c7f536932435c27d4f979dd0640/networks/p2p/message.go#L226-L252

Comment (ehnuje) [reply node/cn/peer_test.go]: Good point! I will add more lines for that.

**Comment by ehnuje** [comment]
@jeonghonah @KimKyungup @nogang Dear reviewers, could you please review this change?

**Comment by ehnuje** [comment]
@KimKyungup @nogang Could you review this change?

File: node/cn/peer_test.go [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+}
+
+func TestBasePeer_AsyncSendTransactions(t *testing.T) {
+	sentTxs := types.Transactions{tx1}
+	lastTxs := types.Transactions{types.NewTransaction(333, addrs[0], big.NewInt(333), 333, big.NewInt(333), addrs[0][:])}
+
+	basePeer, _, _ := newBasePeer()
+
+	// To queuedTxs be filled with transactions
+	for i := 0; i < 128; i++ {
```
Comment (KimKyungup) [inline node/cn/peer_test.go]:
```suggestion
	for i := 0; i < maxQueuedTxs; i++ {
```

**Comment by ehnuje** [comment]
@jeonghonah @nogang Could you review this change?

**Pull request merged**